### PR TITLE
Use fractional anomalies for precipitation timeseries

### PR DESCRIPTION
**Pull request by fmaussion**

### CODE REVIEW

**Comment by fmaussion** [comment]
@anoukvlug you should consider using this code for your purposes from now on.

This change won't make it for version 1.0 of OGGM, but the CESM part could use this new default. Let's talk about the strategy to use offline in slack

**Comment by anoukvlug** [comment]
Thanks for making these changes @fmaussion. Discussing it further on Slack sounds like a good plan.

**Comment by fmaussion** [comment]
OK this should be ready. @anoukvlug can you have a look? Thanks!

File: docs/mass-balance.rst [hunk 1]
```diff
@@ -70,6 +73,21 @@ folder for an example.
     example_plot_temp_ts()  # the code for these examples is posted below
 
 
+GCM data
+~~~~~~~~
+
+OGGM can also use climate model output to drive the mass-balance model. In
+these case we still rely on gridded observations (CRU) for the baseline
```
Comment (anoukvlug) [inline docs/mass-balance.rst]: "this case" instead of "these case"

**Comment by fmaussion** [comment]
In it goes!

**Pull request merged**